user/incus/template.py

```diff
@@ -0,0 +1,81 @@
+pkgname = "incus"
+pkgver = "6.4.0"
+pkgrel = 10
+build_style = "go"
+make_build_args = ["./..."]
+hostmakedepends = [
+    "automake",
+    "libtool",
+    "bash",
+    "git",
+    "go",
+    "go-md2man",
+    "less",
+    "pkgconf",
+]
+makedepends = [
+    "acl-devel",
+    "cowsql-devel",
+    "libseccomp-devel",
+    "libcap-devel",
+    "linux-headers",
+    "libuv-devel",
+    "lxc-devel",
+    "raft-devel",
+    "sqlite-devel",
+    "udev-devel"
+]
+depends = [
+    "libvirt",
+    "acl",
+    "acl-progs",
+    "lxc",
+    "rsync",
+    "squashfs-tools",
+    "xz",
+    "dnsmasq",
+    "iptables",
+    "attr-progs",
+    "util-linux"
+]
+go_build_tags=[ "libsqlite3" ]
+pkgdesc = "Powerful system container and virtual machine manager"
+maintainer = "tj <tjheeta@gmail.com>"
+license = "Apache-2.0"
+url = "https://github.com/lxc/incus"
+source = f"{url}/archive/v{pkgver}.tar.gz"
+sha256 = "c6ad8260a4954b0e4f77de66220d5450290b57858f8c16ee83f12996eea86db4"
+# void ignores TestConvertNetworkConfig (https://github.com/void-linux/void-packages/blob/master/srcpkgs/incus/template)
+# go test -v -tags libsqlite3 -skip TestConvertNetworkConfig ./...
+options = ["!check"]
+
+
+def post_build(self):
+    # Build the agent statically
+    self.env["CGO_ENABLED"] = "0"
+    self.env["CGO_LDFLAGS"] = "-extldflags=-static"
+    self.go_build_tags = ["agent", "netgo", "osusergo"]
+    self.make_build_args = ["-buildmode=default", "./..."]
+    self.golang.build(wrksrc="cmd/incus-agent")
+
+def do_install(self):
+    self.install_file(f"{self.make_dir}/incusd", "usr/bin")
```

Comment:
the default do_install for go buildstyle installs all bins from make_dir to usr/bin, does that not work here? it should do the same thing

Reply:
There were a bunch of extra binaries that were built, so I only wanted to take the ones that mattered.

Reply:
Not sure if there's an easier way to filter them out.

Reply:
how many and what kind? might make more sense to self.uninstall a few instead

Reply:
yah, there were only a few. i'll make the change.